docs/git-workflow.md

```diff
@@ -56,3 +56,24 @@ $ git rebase jetpack/master
 # Then push your changes to your forked version of Jetpack
 $ git push -f origin update/my-changes
 ```
+
+### Keeping your PR up to date if you have commit access to this repo
```

Comment:
Aren't the instructions the same even if you don't have commit access?

Reply:
>Aren't the instructions the same even if you don't have commit access?

 no you need to add another remote reference (you have one for your fork of Jetpack, and you need another for the main Jetpack repo) .

Reply:
About the link. I think it's healthy to have specific steps for our use case so we can directly link persons to these when reviewing PRs that get stalled or conflicting. Maybe we could improve the texts here.

Reply:
Should we extend your info so it's valid for both folks with commit access and folks without, by mentioning where to fetch changes from?

Reply:
I guess your intention is to unify and just leave one section. My intention was to have a clear set of pasteable commands for folks for which, just working with Git was a big step as a contributor.

Reply:
That makes sense. I think we can have 2 sections. Maybe we can just extend the section you added to make it more detailed, i.e. check the remote you are using and fetch from the right one?